packages/ripple-binary-codec/src/types/currency.ts

```diff
@@ -87,12 +87,13 @@ class Currency extends Hash160 {
     super(byteBuf ?? Currency.XRP.bytes)
     const code = this.bytes.slice(12, 15)
 
-    if (this.bytes[0] !== 0) {
-      this._iso = null
-    } else if (/^0*$/.test(this.bytes.toString('hex'))) {
+    if (/^0*$/.test(this.bytes.toString('hex'))) {
```

Comment:
`this.bytes.toString('hex')` is now used twice - can you make a variable for it so it doesn't need to be recomputed?